src/GRPC/ProtocCommandBuilder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\RoadRunnerBridge\GRPC;
+
+use Spiral\Files\FilesInterface;
+use Spiral\RoadRunnerBridge\Config\GRPCConfig;
+
+/**
+ * @internal
+ */
+final class ProtocCommandBuilder
+{
+    public function __construct(
+        private readonly FilesInterface $files,
+        private readonly GRPCConfig $config,
+        private readonly string $protocBinaryPath
+    ) {
+    }
+
+    public function build(string $protoFile, string $tmpDir): string
```

Comment:
we don't use $protofile in code, only `\dirname`, probably we need to pass instead $protoDir?